nodes/src/nodes/agent_rocketride/rocketride_agent.py

```
@@ -134,7 +135,11 @@ def _run(
                     current_scratch=current_scratch,
                 )
             except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)